WikiWatershed/model-my-watershed
# Replace TR-55 Colors with Accessible Variants

## Overview

These colors work better under color accessibility tests, and can serve a more inclusive set of users. The change is required primarily for the runoff site. MMW colors are updated correspondingly for consistency.

Connects #3359
Connects https://github.com/WikiWatershed/mmw-micro/issues/18

### Demo

![Screen Shot 2020-07-27 at 11 33 42 AM](https://user-images.githubusercontent.com/1430060/88946396-35863200-d25d-11ea-906a-d2dfa6148323.png)

![Screen Shot 2020-07-27 at 11 33 54 AM](https://user-images.githubusercontent.com/1430060/88946419-3b7c1300-d25d-11ea-951f-4b04b65653c2.png)

![Screen Shot 2020-07-27 at 11 34 10 AM](https://user-images.githubusercontent.com/1430060/88946431-4171f400-d25d-11ea-8ef0-e23f36d891e4.png)

### Notes

The above screenshots are taken in Firefox, which has notably more vibrant colors than Chrome on macOS. This is a [known issue](https://stackoverflow.com/a/6338124/6995854). The new colors are accessible on both browsers.

## Testing Instructions

* Check out this branch and `bundle`
* Create a TR-55 project
  - [ ] Ensure the colors are different, and match the work done in https://github.com/WikiWatershed/mmw-micro/pull/27
* Open the compare view
  - [ ] Ensure colors are updated there too
